package.json

```diff
@@ -20,7 +20,7 @@
     "eslint:docs": "eslint --fix docs/src/documentation --report-unused-disable-directives",
     "loki:approve": "yarn components loki approve",
     "loki:test": "yarn components loki test",
-    "orbit-kiwi:build": "yarn && yarn tokens build && yarn components build && yarn themer build && yarn docs build",
+    "orbit-kiwi:build": "yarn tokens build && yarn components build && yarn themer build && yarn tailwind-preset build && yarn docs build",
```

Comment:
I think we should build the tailwind-preset _before_ the components. Because the components' build script relies on the built tailwind-preset.